packages/react-dom/src/__tests__/ReactRenderDocument-test.js

```diff
@@ -54,15 +54,15 @@ describe('rendering React components at document', () => {
       const body = testDocument.body;
 
       ReactDOM.hydrate(<Root hello="world" />, testDocument);
-      expect(testDocument.body.innerHTML).toBe('Hello world');
+      expect(testDocument.body.textContent).toBe('Hello world');
 
       ReactDOM.hydrate(<Root hello="moon" />, testDocument);
-      expect(testDocument.body.innerHTML).toBe('Hello moon');
+      expect(testDocument.body.textContent).toBe('Hello moon');
 
       expect(body === testDocument.body).toBe(true);
     });
 
-    it('should not be able to unmount component from document node', () => {
+    it('should be able to unmount component from document node, but leaves persistent nodes intact', () => {
```

Comment:
This test was incorrectly described for 5 years. it says you shouldn't be able to do something but it asserts that you can (b/c fiber). I've updated the test description and modified it based on the new constraints of insertion markers for head/body